lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2757,6 +2773,67 @@ class ConvertAtenOnesOp : public OpConversionPattern<AtenOnesOp> {
 };
 } // namespace
 
+namespace {
```

Comment:
The `ConvertElementWiseOp` should cover this already so this pattern is not really needed. You can refer to other elementwise ops https://github.com/llvm/torch-mlir/blob/127c7d8e27ede211af77392b5277ba06fb515fb7/lib/Conversion/TorchToLinalg/TorchToLinalg.cpp#L1655